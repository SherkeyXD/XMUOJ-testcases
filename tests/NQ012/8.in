350 376
529 676
885 986
420 664
292 525
550 625
685 895
500 540
857 901
325 810
730 900
214 784
160 230
932 988
809 927
442 640
232 707
708 748
590 890
223 265
667 974
643 691
931 991
200 992
390 814
580 753
928 954
894 962
616 944
260 625
835 914
580 619
484 484
662 673
144 312
364 934
548 751
310 925
312 441
910 988
304 780
304 569
724 987
296 408
700 712
895 943
598 728
174 769
104 260
660 850
754 976
796 990
991 994
221 629
653 912
136 891
624 688
960 965
280 839
676 875
215 584
795 986
440 980
973 985
850 948
356 863
397 589
882 938
925 994
667 857
