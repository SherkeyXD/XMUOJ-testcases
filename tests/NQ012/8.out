370 371
no
no
no
370 371 407
no
no
no
no
370 371 407
no
370 371 407
no
no
no
no
370 371 407
no
no
no
no
no
no
370 371 407
407
no
no
no
no
370 371 407
no
no
no
no
153
370 371 407
no
370 371 407
370 371 407
no
370 371 407
370 371 407
no
370 371 407
no
no
no
370 371 407
153
no
no
no
no
370 371 407
no
153 370 371 407
no
no
370 371 407
no
370 371 407
no
no
no
no
370 371 407
407
no
no
no
